eng/jacoco-test-coverage/pom.xml

```diff
@@ -64,7 +64,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-communication-common</artifactId>
-      <version>1.1.0-beta.1</version> <!-- {x-version-update;com.azure:azure-communication-common;current} -->
+      <version>1.0.3</version> <!-- {x-version-update;com.azure:azure-communication-common;dependency} -->
```

Comment:
Why do you need to change the version here?

Reply:
I'd have thought that the versions should match, but if that's not the case I can revert to its original values.

Reply:
I don't believe you need this change here. The other changes look correct and should be enough,

Reply:
Removed.